config/helm/eventing-auth-manager/templates/metrics-reader-rbac.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Why do we need this ClusterRole?

Reply:
Good question! However, it's not worked with just Role/RoleBinding. I think, it needs ClusterRole to retrieve the resources from the cache.